parity/rpc_apis.rs

```diff
@@ -49,6 +49,8 @@ pub enum Api {
 	Net,
 	/// Eth (Safe)
 	Eth,
+	/// Evm (Safe)
```

Comment:
This API set is definitely unsafe, I don't think we should allow dapps to have access to those methods by default.
At best it should be enabled by default only when running with `--chain=dev`, but not for other chains.